ios/FluentUI/Badge Field/BadgeField.swift

```diff
@@ -453,7 +453,9 @@ open class BadgeField: UIView {
             calculateBadgeFrame(badge: badge, badgeIndex: index, lineIndex: &lineIndex, left: &left, topMargin: 0, boundingWidth: bounds.width)
         }
 
-        if isEditable && isFirstResponder && left + Constants.textFieldMinWidth > boundingWidth {
+        let isFirstResponderOrHasTextFieldContent = isFirstResponder || (!isFirstResponder && !textFieldContent.isEmpty)
```

Comment:
if the condition is just (isFirstResponder || !textFieldContent.isEmpty), we'd have the same results, right?

If we're evaluating the right side of the || operation, it already means that isFirstResponder is false.